AppCreationScripts/apps.json

```diff
@@ -0,0 +1,35 @@
+{
+	"Sample": {
+		"Title": "Calling Microsoft Graph from a macOS app",
+		"Level": 400,
+		"Client": "macOS"
+	},
+	"AppRegistrations": [{
+		"x-ms-id": "active-directory-macOS-swift",
+		"x-ms-name": "macOS-swift",
+		"x-ms-version": "2.0",
+		"replyUrlsWithType": [{
+			"url": "msal{AppId}://auth",
```

Comment:
this doesn't look right. Redirect URI format should be `msauth.<bundleId>://auth` for both iOS and Mac

Reply:
Good catch, i copied this part wrong from iOS. Will update both on both platform

Reply:
Updated